datacapturing/src/main/AndroidManifest.xml

```diff
@@ -15,11 +16,19 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
 
     <application>
-        <!-- Declares the data capturing service which is not exported to other applications. Most important is the android:process declaration with the ":" in the beginning. It tells the system to run the service in its own process. -->
+        <!-- Declares the data capturing service which is not exported to other applications.
+        Most important is the android:process declaration with the ":" in the beginning.
+        It tells the system to run the service in its own process. -->
```

Comment:
As the auto-format is not configured to break this comment down I did this manually.